src/Cache/RedisCacheTagsChecksum.php

```diff
@@ -116,21 +134,8 @@ public function calculateChecksum(array $tags) {
    * {@inheritdoc}
    */
   public function reset() {
-    $this->tagCache = array();
-    $this->invalidatedTags = array();
-  }
-
-  /**
-   * Return the key for the given cache tag.
-   *
-   * @param string $tag
-   *   The cache tag.
-   *
-   * @return string
-   *   The prefixed cache tag.
-   */
-  protected function getTagKey($tag) {
```

Comment:
Really no need of a function to get rid of a single line of code, it's being used only twice in the whole code, let's await for the third use case to popup before trying to factorize this. Premature factorization comes at the price of less readible, less maintainable, and more circonvoluted code.